PovioKit/Classes/Networking/RestClient/RestClient.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  RestClient.swift
+//  PovioKit
+//
+//  Created by Borut Tomažin on 14/01/2019.
+//  Copyright © 2019 Povio Labs. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
+public class RestClient {
+  private let manager: Alamofire.SessionManager
```

Comment:
Can we try the solution without `Alamofire` only using `UrlHttpSession` instead?